joinmarket/support.py

```diff
@@ -294,7 +294,7 @@ def choose_orders(db, cj_amount, n, chooseOrdersBy, ignored_makers=None):
     else:
         orders_fees = sorted(orders_fees, key=feekey)  #sort by ascending cjfee
 
-    log.debug('considered orders = \n' + '\n'.join([str(o) for o in orders_fees
+    log.info('considered orders = \n' + '\n'.join([str(o) for o in orders_fees
```

Comment:
I think keep the considered orders as debug? Seems like the class of stuff that is filling up the terminal a bit too much.